package.json

```diff
@@ -0,0 +1,90 @@
+{
+  "name": "habits-together",
+  "version": "0",
+  "private": true,
+  "main": "expo-router/entry",
+  "scripts": {
+    "start": "cross-env EXPO_NO_DOTENV=1 expo start",
+    "prebuild": "cross-env EXPO_NO_DOTENV=1 pnpm expo prebuild",
+    "android": "cross-env EXPO_NO_DOTENV=1 expo run:android",
```

Comment:
`"android:device": "cross-env EXPO_NO_DOTENV=1 expo run:android --device",`

add this pls